cfe_internal/enterprise/federation/federation.cf

```diff
@@ -208,15 +217,58 @@ bundle agent transport_user
       handle => "ssh_config_configured",
       edit_line => default:insert_lines("IdentityFile $(ssh_priv_key)");
 
-    selinux_enabled.incorrect_ssh_context::
+    selinux_enabled.incorrect_ssh_context.selinux_makefile_present::
+      "$(ssh_home_te_file)"
+        create => "true",
+        edit_defaults => default:empty,
+        edit_line => default:insert_lines("module cfenigne-cftransport 1.0;
```

Comment:
MMM, edit_line so that it's compatible with 3.7 which didn't have inline_mustache? I might leave some kind of comment about TODO refactor to use edit_template inline_mustache